.env

```diff
@@ -0,0 +1,3 @@
+DATASTORE_PROJECT_ID=test-project
+DATASTORE_PORT=8080
+DATASTORE_HOST=localhost
```

Comment:
This is somewhat related to Soonil's comment about committing API keys. I believe Ved mentioned putting the API key in this .env file or another file that won't get pushed to github. I believe this file is not supposed to be added to github.

Reply:
Oh I see, I'll take that off. I can't happen to find the .env file anywhere on cloudshell